server/src/main/java/com/cloud/consoleproxy/ConsoleProxyManager.java

```diff
@@ -31,9 +33,14 @@
     public static final int DEFAULT_PROXY_URL_PORT = 80;
     public static final int DEFAULT_PROXY_SESSION_TIMEOUT = 300000;        // 5 minutes
 
+    public static final int DEFAULT_NOVNC_PORT = 8080;
+
     public static final String ALERT_SUBJECT = "proxy-alert";
     public static final String CERTIFICATE_NAME = "CPVMCertificate";
 
+    public static final ConfigKey<Boolean> NoVncConsoleDefault = new ConfigKey<Boolean>("Advanced", Boolean.class, "novnc.console.default", "true",
+        "If true, noVNC console will be default console for virtual machines", true);
```

Comment:
I think it should be disabled by default at least until all hypervisors tested working properly

Reply:
I've tested it on KVM, VMware and Xen and it works fine!